src/tests/electron/tests/android-setup-detect-service-spinner.test.ts

```diff
@@ -10,13 +10,16 @@ import {
     simulateServiceNotInstalled,
 } from '../../miscellaneous/mock-adb/setup-mock-adb';
 
-describe('Android setup - detect-service', () => {
+const description = 'detect-service-spinner';
+describe(`Android setup - ${description}`, () => {
     const defaultDeviceConfig = commonAdbConfigs['single-device'];
     let app: AppController;
 
     beforeEach(async () => {
         await setupMockAdb(
             delayAllCommands(3000, simulateServiceNotInstalled(defaultDeviceConfig)),
+            description,
+            'beforeEach',
```

Comment:
I like the concept of isolating the results per test suite. Completing that process, though, feels like there ought to be some sort of a "reset to default", so that one suite can't "leak" into another. Maybe a `resetMockAdb` function that goes into `afterEach` or `afterAll` ?

Reply:
done via global-teardown